source/user-manual/user-administration/single-sign-on/azure-active-directory.rst

```diff
@@ -138,7 +138,7 @@ Azure Active Directory Configuration
 
    - In option 3 **SAML Certificate**, the **App Federation Metadata Url** will be the ``idp.metadata_url`` in the Wazuh indexer configuration file.
 
-   - Go to the metadata URL using your web browser and note the **X509Certificate**. It will be our ``exchange_key``:
+   - Go to the metadata URL using your web browser, Copy the blob of the **X509Certificate** excluding the ``<X509Certificate>`` and ``<X509Certificate>`` fields. It will be our ``exchange_key``:
```

Comment:
```suggestion
   - Go to the metadata URL using your web browser. Copy the blob of the **X509Certificate** excluding the ``<X509Certificate>`` and ``<X509Certificate>`` fields. It will be our ``exchange_key``:
```